src/utils/operationClass.js

```diff
@@ -0,0 +1,8 @@
+class Operation {
+  constructor(index, expression) {
+    this.index = index;
+    this.expression = expression;
+  }
+}
+
+export default Operation;
```

Comment:
I'm not sure of this, let's disccus it!

Reply:
if it is about the "index" nomeclature I'll change it, I thought I changed it to "id", sorry